consumption.tex

```diff
@@ -0,0 +1,50 @@
+\section{Consumption}
+
+We describe how the target blockchain can verify a burn which happened on the source blockchain. Specifically, we will study the case of Bitcoin and Ethereum as source blockchains. Most cryptocurrencies on the market are adaptations of either of these. We define a party which we call a \emph{prover} who is responsible for constructing a proof of the event and sending it to the target blockchain. The proof we need to send is one of transaction inclusion in the honestly adopted chain of the source blockchain. We will denote this proof as a triple $\beta = \sf(txRaw, txInclusion, blockHeader, blockConnection)$.
+
+\subsection{Transaction Inclusion Proof}
+
+For simplicity we will abbreviate the transaction inclusion part of the complete proof as $\alpha = \sf(txRaw, txInclusion, blockHeader)$.
+
+\import{./}{algorithms/alg.verify-tx.tex}
+
+The transaction inclusion proof is conveniently defined by both Bitcoin and Ethereum. In Bitcoin, each block header contains a commitment to the set of the transaction ids in the block in the form of a Merkle Tree root. Thus, for Bitcoin our transaction inclusion proof is of the form $\mathsf{txInclusion} = (\mathsf{txIDMerkleRoot}, \mathsf{txIDMerkleProof})$. The shape of the proof is as follows $\mathsf{txIDMerkleProof} = ([x_0, x_1, \dots], \mathsf{txIndex}, \mathsf{txCount})$.
```

Comment:
Formulae must fit in the narrative of the text. "As follows" needs to be followed by ":", as if the formula were a word.